tests/postgres_tests/test_unaccent.py

```diff
@@ -42,6 +43,24 @@ def test_unaccent_chained(self):
             ordered=False
         )
 
+    def test_unaccent_chained_with_conforming_strings_off(self):
+        """
+        Chained unaccent causes the comparison string to be escaped.
+        Make sure the escaping SQL is valid even if the DB has standard_conforming_strings OFF.
+        """
+
+        with connection.cursor() as cur:
+            cur.execute("SET standard_conforming_strings TO OFF;")
+            try:
+                self.assertQuerysetEqual(
+                    self.Model.objects.filter(field__unaccent__endswith="éÖ"),
+                    ["àéÖ", "aeO"],
+                    transform=lambda instance: instance.field,
+                    ordered=False)
+            except DatabaseError:
+                self.fail("DatabaseError should not have occurred.")
+            cur.execute("SET standard_conforming_strings TO ON;")
```

Comment:
This should be:
```python
try:
    cursor.execute('SET standard_conforming_strings TO ON;')
    ...
finally:
    cursor.execute('...')
```
so that if the assertion fails, the "tear down" in finally still happens. The second query shouldn't assume the original value of standard_conforming_strings is "on".

Reply:
@timgraham is there a way to get the original value of standard_conforming_strings so that we can set it back to original?